pwndbg/__init__.py

```diff
@@ -72,6 +73,12 @@
 except:
     pass
 
+# this is an unconventional workaround to
+# support unicode printing for python2
+# https://github.com/pwndbg/pwndbg/issues/117
+if six.PY2:
+    reload(sys)
+    sys.setdefaultencoding('utf-8')
```

Comment:
Hmm, I was assuming the escape character wasn't valid UTF-8.  It looks like `\x1b` is fine with UTF-8.